### ISSUE TITLE
connection.query is not a function

### ISSUE THREAD

**Issue by RWOverdijk** [trailsjs/trailpack-waterline#85]
When using a new app (trails) with express (5) and waterline (pack), trying to create a new record gives me connection.query is not a function.

The tables (migrations) get created perfectly. So the connection does work. Why don't queries work?

Update: This is using sails-mysql
Update 2: After logging [connection](https://github.com/balderdashy/sails-mysql/blob/master/lib/adapter.js#L410), I noticed it's a switchback. Perhaps a version compatibility problem?

**Comment by TareqElMasriDev** [comment]
+1

**Comment by cardimajs** [comment]
+1

i have same problem
any solution to this?

**Comment by RWOverdijk** [comment]
Yes, use Wetland ORM https://github.com/SpoonX/trailpack-wetland :D
